tools/PI/DevHome.PI/Helpers/InternalToolsHelper.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+
+namespace DevHome.PI.Helpers;
+
+internal sealed class InternalToolsHelper
+{
+    public static readonly InternalToolsHelper Instance = new();
+
+    private readonly List<Tool> _allInternalTools;
+
+    public ReadOnlyCollection<Tool> AllInternalTools => _allInternalTools.AsReadOnly();
+
+    private InternalToolsHelper()
+    {
+        // All internal tools should be in this list
```

Comment:
Can the comment include the location to keep in sync with?  Something like "This needs to stay in sync with all the internal tools.   The list is at X.  When the list is modified, this needs to reflect the changes."

Reply:
There isn't another list of internal tools anywhere... just as internal tools are added to the product, this is where we should add them to make them available to users.

Reply:
If I add a new tool but forget to add it to the list, what happens, the tool won't show up to users?